README.md

```diff
@@ -71,6 +71,10 @@ https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Projekte_RKI/Nowcas
 
 https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Daten/Testzahlen-gesamt.xlsx?__blob=publicationFile
 
+**Incidence data**
```

Comment:
That should be `**Frozen-Incidence data**`
the other updating incidences are calculated from weekly new cases